src/Microsoft.SqlTools.ServiceLayer/Agent/Common/AgentUtilities.cs

```diff
@@ -82,9 +78,9 @@ public static AgentNotebookInfo ConvertToAgentNotebookInfo(JobProperties job)
                 Category = job.Category,
                 CategoryId = job.CategoryID,
                 CategoryType = job.CategoryType,
-                LastRun = job.LastRun != null ? job.LastRun.ToString() : string.Empty,
-                NextRun = job.NextRun != null ? job.NextRun.ToString() : string.Empty,
-                JobId = job.JobID != null ? job.JobID.ToString() : null,
+                LastRun = job.LastRun.ToString(),
+                NextRun = job.NextRun.ToString(),
+                JobId = job.JobID.ToString(),
```

Comment:
> ##[warning]src\Microsoft.SqlTools.ServiceLayer\Agent\Common\AgentUtilities.cs(85,27): Warning CS8073: The result of the expression is always 'true' since a value of type 'DateTime' is never equal to 'null' of type 'DateTime?'

> ##[warning]src\Microsoft.SqlTools.ServiceLayer\Agent\Common\AgentUtilities.cs(86,27): Warning CS8073: The result of the expression is always 'true' since a value of type 'DateTime' is never equal to 'null' of type 'DateTime?'

> ##[warning]src\Microsoft.SqlTools.ServiceLayer\Agent\Common\AgentUtilities.cs(87,25): Warning CS8073: The result of the expression is always 'true' since a value of type 'Guid' is never equal to 'null' of type 'Guid?'